state-chain/pallets/cf-ingress-egress/src/lib.rs

```diff
@@ -1144,6 +1155,25 @@ pub mod pallet {
 			})?;
 			Ok(())
 		}
+
+		/// Updates the max swap retry duration .
+		///
+		/// ## Events
+		///
+		/// - [MaxSwapRetryDurationBlocks](Event::MaxSwapRetryDurationBlocks)
+		#[pallet::call_index(10)]
+		#[pallet::weight(T::WeightInfo::update_max_swap_retry_duration())]
+		pub fn update_swap_retry_delay(
```

Comment:
same here, we can just add this to the update_pallet_config function that exists already